### PR TITLE
Feature/survey validation

### PR DESCRIPTION
**Pull request by utas-tajona**

### CODE REVIEW

**Review by akashisama — CHANGES_REQUESTED** [review]

File: api/src/main/java/au/org/aodn/nrmn/restapi/validation/BaseFormattedValidator.java [hunk 1]
```diff
@@ -12,5 +12,5 @@ public BaseFormattedValidator(String columnTarget) {
         this.columnTarget = columnTarget;
     }
 
-    abstract public <T> Validated<StagedRowError, T> valid(StagedRowFormatted target);
+    abstract public  Validated<StagedRowError, String> valid(StagedRowFormatted target);
```
Comment (akashisama) [inline api/src/main/java/au/org/aodn/nrmn/restapi/validation/BaseFormattedValidator.java:15]: unneeded space between public and Validated

**Comment by akashisama** [comment]
Don't forget to update the backlog item with the tasks your doing and PRs that need to be reviewed

File: api/src/main/java/au/org/aodn/nrmn/restapi/validation/BaseFormattedValidator.java (hunk 1 above)
Comment (utas-tajona) [reply api/src/main/java/au/org/aodn/nrmn/restapi/validation/BaseFormattedValidator.java]: space out

File: api/scripts/migration_schema.sh [hunk 2]
```diff
@@ -11,8 +11,7 @@ PROJECT_DIR="$(cd "$(dirname "$0")"/..; pwd -P)"
 
 echo "Updating migration.sql from 17-nec-hob..."
 
-pg_dump -v -h 17-nec-hob.emii.org.au -p 5432 --no-acl --no-owner --schema nrmn \
-  --schema-only nrmn_migration --disable-dollar-quoting > "$PROJECT_DIR/src/main/resources/sql/migration.sql"
+pg  --schema-only nrmn_migration --disable-dollar-quoting > "$PROJECT_DIR/src/main/resources/sql/migration.sql"
```
[the remark this answers is not in the document]
Comment (utas-tajona) [reply api/scripts/migration_schema.sh]: removed

File: api/src/main/java/au/org/aodn/nrmn/restapi/validation/provider/ValidatorProvider.java [hunk 3]
```diff
@@ -13,6 +13,6 @@
         F extends BaseFormattedValidator,
         G extends  BaseGlobalValidator> {
   Seq<Tuple2<String, R>> getRowValidators();
-  Seq<F> getFormattedValidators();
+  Seq< F> getFormattedValidators();
```
[the remark this answers is not in the document]
Comment (utas-tajona) [reply api/src/main/java/au/org/aodn/nrmn/restapi/validation/provider/ValidatorProvider.java]: space out

**Review by akashisama — APPROVED** [review]

**Pull request merged**